draft-ietf-rats-architecture.md

```diff
@@ -844,40 +835,89 @@ a different format.
 state of the Attester, and the Appraisal Policy for Evidence,
-might change immediately after the Evidence or Attestation Result was generated.
-The goal is merely to narrow the time window to something the Verifier
+might change immediately after the Evidence or Attestation Results were generated.
+Freshness is a key component for enabling caching
+and reuse of both Evidence and Attestation Results, which is especially
+valuable in cases where their computation uses a substantial part of a
+resource budget (e.g., an energy budget in constrained devices).
+The goal is to narrow the time window to something the Verifier
 (for Evidence) or Relying Party (for an Attestation Result) is willing to accept.
 
-There are two common approaches to providing some assurance of freshness.
-The first approach is that a nonce is generated by a remote entity
+There are four common approaches to providing some assurance of freshness, effectively determining the epoch of the
+validity-span of, e.g., Evidence or Attestation Results.
+
+## Implicit Timekeeping using Nonces
+
+The first approach places the onus of timekeeping solely on the remote
+entity (e.g., the Verifier for Evidence, or the Relying Party for an
+Attestation Result).
+This might be an appropriate choice in case the Attester does not have a reliable clock or time synchronisation is
+otherwise impaired.
+In this approach, a nonce is generated by a remote entity
 (e.g., the Verifier for Evidence, or the Relying Party for an Attestation
 Result), and the nonce is then signed and included along with the claims in
 the Evidence or Attestation Result, so that the remote entity knows that the
 claims were signed after the nonce was generated.
 
+## Explicit Timekeeping using Synchronized Clocks
+
 A second approach is to rely on synchronized clocks, and include a signed
-timestamp (e.g., using {{-rats-tuda}}) along with the
+timestamp along with the
 claims in the Evidence or Attestation Result, so that the remote entity knows
 that the claims were signed at that time, as long as it has some assurance that
-the timestamp is correct.  This typically requires additional claims about
+the timestamp is correct.
+This typically requires additional claims about
 the signer's time synchronization mechanism in order to provide such assurance.
-
-In either approach, it is important to note that the actual values in claims
-might have been generated long before the claims are signed.  If so, it is the
-signer's responsibility to ensure that the values are still correct when they
-are signed.  For example, values might have been generated at boot, and then
-used in claims as long as the signer can guarantee that they cannot have changed
+This allows associating a "rough" epoch to the Evidence or Attestation Result. 
+In this case, the epoch is said to be "rough" because 
+    a) it is always bound to the set of Claims as a whole, as opposed to having Claim being separately annotated, and 
```

Comment:
"having Claim" is bad grammar.
"having Claims"?  "having a Claim"? "having each Claim"?

But which approach is the one where each Claim can be separately annotated with a timestamp?   In the original text with only two approaches, it would be part of this one.

Reply:
This has been addressed using Ned's suggestion in c215569